pkgs/by-name/me/meson/package.nix

```diff
@@ -118,7 +118,7 @@ python3.pkgs.buildPythonApplication rec {
     ''test cases/osx/7 bitcode''
   ])
   ++ [
-    ''HOME="$TMPDIR" python ./run_project_tests.py''
+    ''HOME="$TMPDIR" ${python3.interpreter} ./run_project_tests.py''
```

Comment:
Shouldn’t this be using `python3.executable` instead to support cross-compilation?
I can’t really test this though because sadly pypy itself is broken :(

Reply:
> Shouldn’t this be using `python3.executable` instead to support cross-compilation? I can’t really test this
I dont know which would be better, `python3.interpreter` worked for me

Reply:
The package already has  `python3.interpreter` at some other places, we i think it would make sense to use it too here

![image](https://github.com/NixOS/nixpkgs/assets/53050011/d7ffc057-7a80-4b50-99aa-c8ac539ebd67)

Reply:
Ah, that’s in the `checkPhase`, should be fine-ish then, unless someone takes the herculean effort of making `doCheck = true` by default in stdenv instead of always disabling it for build ≠ host case.
FWIW `python.interpreter` is an alias for `${python}/bin/${executable}`, so I *think* using `executable` would be better, assuming that `python` is already in PATH/nativeBuildInputs. Otherwise LGTM 👍